### ISSUE TITLE
Router commands

### ISSUE THREAD

**Issue by unwriter** [interplanaria/bit#1]
There are two proposed commands that need to be added to [Bitcom](https://bitcom.bitdb.network) in order to make Bit:// work.

- `route enable`: for the protocol admin to open up routes that service providers can plug into
- `route add`: a command for service providers to plug into the global router

Here's the relevant section: https://bit.planaria.network/#/?id=c-implementation

This format needs to be support as wide variety of use cases as possible while keeping simplicity.

Ask questions and share feedback.

**Comment by deggenschwiler** [comment]
Am I right in thinking that, once written - the name and description are set in stone? I see that calls to the same echo name make no difference, as the protocol is set by the first call to that expression?

Also curious - is the plan for Bottle browser to make bit:// requests natively?

**Comment by unwriter** [comment]
> Am I right in thinking that, once written - the name and description are set in stone?

No, the most recent one is the lastest version. As you can see here i tried updating the name three times, and the final version is displayed as the official name https://bterm.network/#address=1KCm9cgDdT7r88WRQmm6gynFNmTrskdygn&out.s5=name

The caveat at the moment is it can't distinguish between transactions that happened within the same block (which means if you update the name multiple times within the same block the current version won't be able to tell which is the latest version), but I'm working on a Planaria feature update that will be able to order the txs within the same block, so that will be solved too.

> Also curious - is the plan for Bottle browser to make bit:// requests natively?

Yes. I'm working on this right now.
